lib/elixir/lib/kernel.ex

```diff
@@ -2805,9 +2805,10 @@ defmodule Kernel do
   ## Module names
 
   A module name can be any atom, but Elixir provides a special syntax which is
-  usually used for module names. What is called a module name is an upper case
-  character followed by any alphanumeric characters or `_` or `.`. This
-  identifier is equivilant to an atom prefixed by `Elixir.`. So in the
+  usually used for module names. What is called a module name is an _unaccented
+  uppercase Latin letter_ followed by any number of _unaccented lowercase or
```

Comment:
I am concerned that latin letters would leave it up to the interpretation that accents are allowed. Specially because they are allowed in Erlang. Using ascii is a bit clearer I think.